compass_metrics_model/utils.py

```diff
@@ -178,7 +175,6 @@ def get_activity_score(item, level="repo"):
             "commit_frequency": [COMMIT_FREQUENCY_WEIGHT_ACTIVITY, COMMIT_FREQUENCY_MULTIPLE_THRESHOLD_ACTIVITY],
             "updated_since": [UPDATED_SINCE_WEIGHT_ACTIVITY, UPDATED_SINCE_MULTIPLE_THRESHOLD_ACTIVITY],
             "org_count": [ORG_COUNT_WEIGHT_ACTIVITY, ORG_COUNT_MULTIPLE_THRESHOLD_ACTIVITY],
-            "created_since": [CREATED_SINCE_WEIGHT_ACTIVITY, CREATED_SINCE_MULTIPLE_THRESHOLD_ACTIVITY],
```

Comment:
instead of remove it directly, could you comment it out? See the differences for now.